BraveShared/Analytics/CustomHeaderData.swift

```diff
@@ -45,7 +45,7 @@ class CustomHeaderData: NSObject {
         var customHeaders: [CustomHeaderData] = []
 
         for (_, object) in json {
-            guard let header = object["headers"].first else { continue }
+            guard let header: (String, JSON) = object["headers"].first, header.0 == "X-Brave-Partner" else { continue }
```

Comment:
Maybe abstract the `X-Brave-Partner` into a constant somewhere, just easier to see what's going on then. Can be in this file, just nice to have broken up.